security/fma-withdrawals-root-in-block-header.md

```diff
@@ -0,0 +1,90 @@
+- [monorepo PR](https://github.com/ethereum-optimism/optimism/pull/13962)
+
+## Failure Modes and Recovery Paths
+
+### FM1: Withdrawals downtime do to inaccurate `withdrawalsRoot`
+
+- **Description:** 
+  If the `withdrawalsRoot` in the block header is incorrect, critical infra used to enable withdrawals may fail. Namely, output proposals and challenges would be incorrect, affecting chains with permissioned and chains with permissionless proofs. This is because these components will, with the activation of the Isthmus fork, use the `withdrawalsRoot` header field instead of querying the information from an archive node in the usual way. Output roots are returned by the op-node [`optimism_outputAtBlock`](https://docs.optimism.io/operators/node-operators/json-rpc#optimism_outputatblock) RPC method, and this behaves differently under Isthmus -- when handling a request for an output root (it no longer delegates an `eth_getProof` to op-geth and just reads the information from the block header).
+  
+  Triggers: 
+  
+  * A failed hardfork activation in the execution client.
+
+  * If there is an execution client bug, for example it is possible the root is (incorrectly) added to the header before the state is fully committed. 
+
+  * If we were to ever introduce non empty `withdrawals` in the block body, this might override the mechanism introduced with this feature and invalidate the interpreation of the `withdrawalsRoot` field. 
+
+- **Risk Assessment:**
+
+  **High impact, low likelihood** 
+  
+  Temporary downtime for withdrawals, or loss of funds if not remediated in time to challenge a malicious proposal.
+
+  **Mitigations:**
+  * We rely on e2e tests to check for consistency between the outputs returned from op-node and those constructed manually in the old way.
+
+  * Instead of waiting for the failure mode to materialize and then writing a patch in a rush, we could add an optional config var to op-node to switch it back into the old behavior. Rolling out the fix would not then require any software releases.
+
+  * op-geth could be modified to log a critical error (triggering an alert) if the withdrawals list in the body is ever non empty.
+
+- **Detection:** 
+  Fault proof monitoring systems may not detect this failure mode immediately, until an actor running patched software made a proposal or challenge.
+
+- **Recovery Path(s)**:
+  Fault proof infra would nee to be pointed at a patched op-node. The patch would restore the old behaviour for generating output roots.
+
+
+## Generic failure modes: 
+See the [generic FMA](./fma-generic-hardfork.md):
+* Chain halt at activation (there is a change to the engine API, which elevates this risk)
+* Activation failure
+* Invalid setImplementation execution
+* Chain split (across clients)
+
+## Specific Action Items
+- [ ] (BLOCKING) e2e tests must check for consistency between output roots returned from op-node and those constructed manually in the old way
```

Comment:
We currently check consistency of withdrawal root itself https://github.com/ethereum-optimism/optimism/blob/6a436fe9ac9acb215b0f4b9f87ccd3832f4d6b72/op-e2e/actions/upgrades/isthmus_fork_test.go#L286-L301 which is probably sufficient